Update README.md (#40)
nimiq/qr-scanner: README.md

```
As an optional third parameter a specific resolution that should be worked on can be specified. The default is 400.

#### 3. Start scanning
```js
qrScanner.start();
```

> Call it when you ready to scan, for example on some button click.

Reviewer: ```suggestion
Call it when you're ready to scan, for example on a button click.
```
